src/utils/images.ts

```diff
@@ -18,3 +24,52 @@ export const getImageDetails = (
     imageDirectory,
   }
 }
+
+/**
+ * Util method to adjust image src in provided HTML, so that they can load from
```

Comment:
nit: this is not just HTML right? includes Mkdwn as well?

Reply:
Hmm this adjustment of the image src is done on the generated HTML provided by `marked.parse()`, so we will be working on the HTML only and not the underlying Markdown.